### ISSUE TITLE
Issue with "Using ARIA: Roles, states, and properties": …

### ISSUE THREAD

**Issue by adbakke** [mdn/content#1439]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques

#### What information was incorrect, unhelpful, or incomplete?
Link to Document Structure Role: Tooltips links to missing page.

#### Specific section or headline?
Document Structure Roles:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#document_structure_roles

#### What did you expect to see?
Expected link to go to existing page (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role)

#### Did you test this? If so, how?
Yes, clicked on the link and received this error when I arrived at the linked page:

> Sorry, the page /en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role could not be found.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/accessibility/aria/aria_techniques/index.html
* Report started: 2021-01-18T15:09:50.178Z

</details>

**Comment by sideshowbarker** [comment]
Unfortunately there are a number of ARIA roles for which we don’t have separate MDN articles written yet — so the “fix” here is to de-link the *tooltip* hyperlink, and some others.

So I opened https://github.com/mdn/content/pull/1522 with a patch that does those de-links — as well as updating some other links to their correct locations.

**Comment by Ryuno-Ki** [comment]
> Unfortunately there are a number of ARIA roles for which we don’t have separate MDN articles written yet

Can we have an issue about the missing articles?
Chris explained me the process in https://github.com/mdn/content/issues/1506#issuecomment-763546942 today.

**Comment by chrisdavidmills** [comment]
Looks like this is already fixed, so closing.
